pkg/auth/dependency/session/context.go

```diff
@@ -1,5 +1,38 @@
 package session
 
+import (
+	"context"
+
+	"github.com/skygeario/skygear-server/pkg/core/auth/authinfo"
+)
+
 type Context interface {
 	Session() *Session
+	User() *authinfo.AuthInfo
```

Comment:
Are we going to replace AuthContextGetter with this interface?

Reply:
Yes, I planned to have three interface for auth contexts:
- `session.Context`: contains user & session, used in auth gear only (in auth)
- `auth.AccessKeyContext`: contains API key (i.e. client ID), used in needed places (in core)
- `authn.Context`: contains `authn.Info`, used in everywhere (in core)